src/main/views/user-profiles.html

```diff
@@ -104,4 +105,7 @@
       </table>
     </div>
   {% endif %}
+  {%else%}
```

Comment:
Is this empty `{% else %}...{% endif %}` block intentional? Perhaps an accidental leftover?